scripts/install/CreateLti1p3RegistrationSnapshotSchema.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA;
+ *
+ * @author Sergei Mikhailov <sergei.mikhailov@taotesting.com>
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoLti\scripts\install;
+
+use oat\generis\persistence\PersistenceManager;
+use oat\generis\persistence\sql\SchemaProviderInterface;
+use oat\oatbox\extension\script\ScriptAction;
+use oat\taoLti\models\classes\Platform\Repository\Lti1p3RegistrationSnapshotSchemaProvider;
+
+class CreateLti1p3RegistrationSnapshotSchema extends ScriptAction
+{
+    public const OPTION_PERSISTENCE = 'persistence';
+
+    public function __invoke($params = []): void
+    {
+        parent::__invoke($params);
+    }
+
+    protected function provideUsage(): array
+    {
+        return [
+            'prefix' => 'h',
+            'longPrefix' => 'help',
+            'description' => 'Shows this message'
```

Comment:
could you please add a more meaningful description?

Reply:
It is meaningful, if you try running it ;)

```
php index.php '\oat\taoLti\scripts\install\CreateLti1p3RegistrationSnapshotSchema' --helpxdebug.client_host/xdebug.client_port) :-(
Create `lti_platform_registration` table.

  Optional Arguments:
    -p persistence, --persistence persistence (default: "default")
    -h help, --help help
      Shows this message
```

Reply:
manifests were probably initially semi-automatically generated, we don't update the headers there as a rule.